Makefile

```diff
@@ -1,17 +1,17 @@
 include vars.mk
 
-ifeq ($(BUILDTIME),)
-  BUILDTIME := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
+ifeq ($(COMMIT_TIME),)
+  COMMIT_TIME := $(shell git show -s --format="%cd" --date=format:"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
```

Comment:
Do we need/want the date to be in UTC?

(I recall having to go through `date` to get that, but it's been a while, and it's dirty. Perhaps there's an easier way https://github.com/docker/docker-ce-packaging/pull/125)

Reply:
Actually this is just an input format. According to https://github.com/docker/app/blob/master/internal/version.go#L24 this will just be used as input to the function that will return in the final format in https://github.com/docker/app/blob/master/internal/version.go#L32.

Here I had a look at the `FIXME` comment on line 31, but since the file does not exist in the cli, I didn't do the suggested change.